task8-1/task8-1/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace task8_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите целое ненулевое число");
+            int b = int.Parse(Console.ReadLine());
+            int i = -b+1;
+            int summa = 0;
+            while (i < b * b)
```

Comment:
Ошибка. Последнее слагаемое упущено
```suggestion
            while (i <= b * b)
```